rs/tests/nested/src/lib.rs

```diff
@@ -142,18 +135,14 @@ pub fn registration(env: TestEnv) {
 pub fn upgrade_hostos(env: TestEnv) {
     let logger = env.logger();
 
-    let target_version_str = std::env::var("ENV_DEPS__HOSTOS_UPDATE_IMG_VERSION").unwrap();
+    let target_version_str = get_hostos_update_img_version().unwrap();
     let target_version =
         HostosVersion::try_from(target_version_str.trim()).expect("Invalid mainnet hostos version");
 
-    let update_image_url_str = std::env::var("ENV_DEPS__HOSTOS_UPDATE_IMG_URL").unwrap();
-    info!(
-        logger,
-        "HostOS update image URL: '{}'", update_image_url_str
-    );
     let update_image_url =
-        Url::parse(update_image_url_str.trim()).expect("Invalid mainnet hostos update image URL");
-    let update_image_sha256 = std::env::var("ENV_DEPS__HOSTOS_UPDATE_IMG_SHA").unwrap();
+        get_hostos_update_img_url().expect("Invalid mainnet hostos update image URL");
```

Comment:
```suggestion
        get_hostos_update_img_url().expect("Invalid hostos update image URL");
```

This isn't necessarily a mainnet url, right?

Reply:
Thanks! I missed updating the comment in a few places.